### PR TITLE
Close sub-menu on item click

### PR DESCRIPTION
**Pull request by robgonnella**
Addresses issue 23
https://github.com/AlexTorresSk/custom-electron-titlebar/issues/23

### CODE REVIEW

File: src/titlebar.ts [hunk 1]
```diff
@@ -115,16 +115,18 @@ export class Titlebar extends Themebar {
...
+
 	private registerListeners() {
 		this.currentWindow.on(EventType.FOCUS, () => {
 			this.onDidChangeWindowFocus(true);
 			this.onFocus();
 		});
 
-		this.currentWindow.on(EventType.BLUR, () => {
-			this.onDidChangeWindowFocus(false);
-			this.onBlur();
```
Comment (AlexTorresSk) [inline src/titlebar.ts]: These lines blur the window, this causes the color of the title bar to change. It's a likely solution, but I'd have to refocus the window.

Comment (robgonnella) [reply src/titlebar.ts]: Ahh good point, I didn't realize the colors changed on blur. I've updated this to just pass `this.menu.blur` directly to bypass the style changes.

Comment (robgonnella) [reply src/titlebar.ts]: Sorry Alex, I didn't see all of the changes you requested after I force pushed one of the changes to my fork. Feel make changes directly to this MR if you want. Or, if you want to re-add the requested changes, I'll push any updates in new commits. Sorry about that!

Comment (zhch-hong) [reply src/titlebar.ts]: invalid

**Pull request merged**